src/Config/Services.php

```diff
@@ -36,4 +37,16 @@ public static function passwords(bool $getShared = true): Passwords
 
         return new Passwords(config('Auth'));
     }
+
+    /**
+     * JWT Manager.
+     */
+    public static function jwtmanager(bool $getShared = true): JWTManager
```

Comment:
I see why you didn't make the adapters as a service now; I still think they might work better non-statically.

Reply:
If we make the adapter a service, it would not have enough functionality. 
The adapter can only encode and decode JWS; JWT also has JWE, which may need to be supported in the future.